src/patternfly/demos/Page/examples/Page.md

```diff
@@ -17,6 +17,59 @@ Using the `.pf-m-full-height` modifier class on the page component eliminates th
 {{> page-template page-template--id="page-demo-full-height" page-template--modifier="pf-m-full-height"}}
 ```
 
+### Multiple body elements
```

Comment:
I'd expect to see multiple `pf-c-page__main-body` with this description. Should it be "Multiple sidebar body elements" ? Are multiple body elements valid, and if so should we show them?